reviewrot/__init__.py

```diff
@@ -282,6 +287,9 @@ def parse_cli_args(args):
     parser.add_argument('--email', nargs="+",
                         default=None,
                         help='send output to list of email adresses')
+    parser.add_argument('--subject',
```

Comment:
Please add DEFAULT_SUBJECT as a default argument here

Reply:
The problem is that I did not find a way to check if user set the value on command line so the value from config should be used instead.

The correct way would be to get default values from configuration and then load from command line. But that would be a bigger refactoring.

Reply:
I've removed the mention about the default here.